common/service/config/persistence.go

```diff
@@ -34,16 +34,8 @@ const (
 )
 
 // SetMaxQPS sets the MaxQPS value for the given datastore
-func (c *Persistence) SetMaxQPS(key string, qps dynamicconfig.IntPropertyFn) {
-	ds, ok := c.DataStores[key]
-	if !ok {
-		return
-	}
-	if ds.Cassandra != nil {
-		ds.Cassandra.MaxQPS = qps
-		return
-	}
-	ds.SQL.MaxQPS = qps
+func (c *Persistence) SetMaxQPS(service Service, qps dynamicconfig.IntPropertyFn) {
+	service.MaxQPS = qps
```

Comment:
We don't need this function anymore because its just one line - more clear to just set it per service.